packages/stackflow/src/primitive/GlobalInteraction/useGlobalInteraction.ts

```
@@ -57,132 +70,254 @@ export function useGlobalInteraction() {
-            stackRef.current?.style.setProperty("--swipe-back-target", "100%");
-            setSwipeBackState("completing");
-          } else {
-            stackRef.current?.style.setProperty("--swipe-back-target", "0");
-            setSwipeBackState("canceling");
-          }
+  const getSwipeBackEvents = useCallback((props: SwipeBackProps) => {
+    const {
+      swipeBackDisplacementRatioThreshold: displacementRatioThreshold = 0.4,
+      swipeBackVelocityThreshold: velocityThreshold = 1,
+    } = props;
+    const onSwipeStart = useCallbackRef(props.onSwipeBackStart);
```

Biome (2.4.10)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

(lint/correctness/useHookAtTopLevel)